oid.md

```diff
@@ -50,3 +50,11 @@ A LDAP attribute, representing a date of agreement on privacy policy.
 
 - partial OID: `1.2.1`
 - complete OID: `1.3.6.1.4.1.53611.1.2.1`
+
+### 1.2.2
+
+A LDAP attribute, representing any kind of custom attribute. For exemple post codes, internal ids
```

Comment:
It's not "any kind of custom attribute".
It's a numeric, more specifically a positive integer meant to identify a polygonal area.

Reply:
ah... So my definition of it in ldap is wrong. I choose a string repr: `1.3.6.1.4.1.1466.115.121.1.15`

In the other hand, [Bastia (corsica)](https://www.insee.fr/fr/metadonnees/cog/commune/COM2B033-bastia) have an inseeCode named 2B033. 

Another example (still in France): [Bourg-en-bresse](https://www.insee.fr/fr/metadonnees/cog/commune/COM01053-bourg-en-bresse) 01053

Reply:
Good point.
Let's go for a string.